test/test.js

```diff
@@ -1,12 +1,12 @@
 var test = require('tape')
 var fs = require('fs')
 var path = require('path')
-var eol = require('os').EOL
 var bops = require('bops')
 var spectrum = require('csv-spectrum')
 var concat = require('concat-stream')
 var csv = require('..')
 var read = fs.createReadStream
+var eol = '\n'
```

Comment:
On Windows `EOL` is `'\r\n'`, which causes a few tests to fail. Making this `'\n'` across all operating systems fixes this problem.